### DISCUSSION TITLE
Cannot connect to Oracle database since version 3.6.5

### DISCUSSION THREAD

**Discussion by lauhin** [dbeaver/dbeaver#380] [Q&A] [answered]
I updated DBeaver to version 3.6.5. After restarting my computer, I cannot connect to some Oracle databases. I got the following error: ORA-01017: invalid username/password.

Reversed my version back to 3.6.3 and I can connect without a problem.
Can you guys look into this :)

Thanks in advance.

Laurent

**Comment by serge-rider** [comment]
Can't reproduce. 
Do you use SSH tunnel or any proxies? If so check  #373.
Otherwise - is there anything special in your environment? Maybe something is in error/debug logs? (https://github.com/serge-rider/dbeaver/wiki/Log-files)

**Comment by lauhin** [comment, marked as answer]
I attached the error log. [error.txt](https://github.com/serge-rider/dbeaver/files/225317/error.txt)
I reinstalled 3.6.5 and no error this time, maybe a corrupt install. Thanks for replying this.
